pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -167,9 +271,15 @@ func TestPipelineRun_Validate(t *testing.T) {
 				Name: "pipelinelinename",
 			},
 			Spec: v1beta1.PipelineRunSpec{
+				Params: []v1beta1.Param{{
+					Name: "pipeline-words",
+					Value: v1beta1.ArrayOrString{
+						ArrayVal: []string{"hello", "pipeline"},
+					},
+				}},
 				PipelineSpec: &v1beta1.PipelineSpec{
 					Params: []v1beta1.ParamSpec{{
-						Name: "pipeline-words",
+						Name: "default-pipeline-words",
```

Comment:
why did you rename this param?

Reply:
yes, calling it `default` when there's a default field was what was confusing -- yes please rename

Reply:
done